Add eslint and prettier

emilcheva opened the pull request:
Configured linting and formatting

emilcheva committed:
Add eslint and prettier

Configured linting and formatting

steliyan commented on package.json:
```diff
@@ -4,7 +4,10 @@
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write './**/*.{js,json}' --config ./.prettierrc",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix"
```
Does this work without specifying a working dir as in the `lint` script?

steliyan commented on package.json:
```diff
@@ -4,7 +4,10 @@
     "start": "node server",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write './**/*.{js,json}' --config ./.prettierrc",
```
Is the `--config` arg necessary?

emilcheva replied:
not really , I'm removing it ;)

steliyan commented on server/resolvers.js:
```diff
@@ -7,14 +7,15 @@ const resolvers = {
-    },
-  },
+    breedImage: ({ id }, _, { dataSources }) => {
+      /*eslint no-undef: "off"*/
```
Why?

emilcheva replied:
the 'limit' is not defined but still needed as an arg :/

steliyan replied:
So what's expected to happen if limit is not defined?

emilcheva replied:
it's optional in the API anyway, without it, we'll have all the images for the breed returned ... with limi set, we are presetting max to be returned

steliyan replied:
So why not just pass `undefined`/`null`/`-1` or whatever? ESLint is rightly emitting an error as such var is not defined.

emilcheva committed:
Remove work files and config options from prettier in package.json

No need to explicitly specify file types or config file

emilcheva committed:
Fixing eslint issue with 'no-def' function argument

The limit is set as second arg with default value directly in the API's getImagesOfBreed
